packages/pages-shared/metadata-generator/parseRedirects.ts

```diff
@@ -119,6 +119,14 @@ export function parseRedirects(input: string): ParsedRedirects {
 		}
 		seen_paths.add(from);
 
+		if (status === 200 && !to.startsWith("/")) {
```

Comment:
We've got a utility in `validateURL.ts` which can check if a URL is relative. I'd like to re-use that if possible. If significant changes to the `validateUrl()` function would be required, we can re-think, but if there's overlap, let's go for that.

Reply:
Done in https://github.com/cloudflare/workers-sdk/pull/2708/commits/14d5ade2ee5299dd323932071b09e9dcc9c37bdc